### ISSUE TITLE
Does xfs_freeze need to be run before running the create-snapshot?

### ISSUE THREAD

**Issue by kentmw** [CaseyLabs/aws-ec2-ebs-automatic-snapshot-bash#17]
I'm reading how to backup mongo from an ebs volume and it suggests locking the db before doing a snapshot. It also suggests locking the filesystem if there's anything else on the volume. https://docs.mongodb.org/ecosystem/tutorial/backup-and-restore-mongodb-on-amazon-ec2/

I was wondering if there could be a way to trigger a lock and unlock before this cronjob runs. Has anyone run into issues where the snapshots aren't able to be used because there is data corrupted during the snapshot?

**Comment by jstaffans** [comment]
Yes, I just ran into this issue. This script works well with `ext4` but restoring snapshots of `xfs` volumes will not work. Should be an easy addition, though.
